lib/discourse_client.rb

```diff
@@ -0,0 +1,18 @@
+require 'discourse_api'
+
+class DiscourseClient < DiscourseApi::Client
```

Comment:
Chcel som aby bol Discourse notifier zavisli iba na interface injektnuteho klienta. A tento koncept narusal fakt, ze musim chytat custom exception class z discourse gemu. Tak som urobil nas discource klient, ktory momentalne len wrapne ten z gemu a takisto prelozi ich exception na exception nasho klienta.

A tu exception potrebujem na to, aby som vedel odfiltrovat discourse api  errory od ostatnych.